### ISSUE TITLE
Review the alt text for Wins page images and ensure that they adhere to WCAG

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2569]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will tackle the images under the Wins page.

#### Details
Developers identified that some alt texts in the Wins page should be reviewed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] Make a copy of the tab Template [Page Name]- [Status] 
- [x] With the copied tab replace [Page Name] with the page you are reviewing and [Status] with "Audit in Progress". Don't forget to remove brackets "[ ]"
- [x] Add a link of the page your working on to the Resources/Instructions section of this issue at the bottom where it says "Google sheet Tab for your page".
- [x] For each image on the Wins page identify if the alt text needs to be changed, or not, using the recourses below, and fill out the google sheet.
#### Summary
If images provide more meaning = needs alt text
If images are decorative = null alt text (alt="")
- [x] When completed change your google sheet tabs status to Audit Complete
- [x] Review with Design Lead or another designer/researcher
- [x] Mark issue as done

Development
- [ ]  Create issues for that particular alt text should be created (preferably labelled good first issue) for new developers to fix

### Resources/Instructions
[Wins page](https://www.hackforla.org/wins/)
[Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
[Decorative Images and alt text](https://www.w3.org/WAI/tutorials/images/decorative/)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Google sheet instructions](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
[Google sheet template](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=129703697)
[Google sheet Tab for your page](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit?pli=1#gid=2134006925)

**Comment by iilashi** [comment]
Progress: Waiting for team review
Blockers: None
Availability: 
ETA: Complete

**Comment by IsaacDesigns** [comment]
@ilashi Issue looks good to me, good job!

**Comment by SAUMILDHANKAR** [comment]
Icon should not have alt text.

**Comment by 7kram** [comment]
We need accessibility testing on the "skills" icons to determine if the speech bubbles work with people with visual acuity issues. 

If yes, we will remove all alt text for these "skills" icons

If no, recommend a course of action

**Comment by ExperimentsInHonesty** [comment]
This issue is ready for dev leads to make the issue to change the alt text on the wins page

1. Turns out tooltips do not replace alt text.  Alt text is still needed on the main wins page, but will have to disappear when you go to the see more (because then the same text appears next to the image).

To experience what It's like for a blind person, you can install this [screen reader chrome extension](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn/related?hl=en)  Please note, you should install in a browser window with only empty tab open or it will try to read all tabs you have open

2. SVG get aria tags, so this work could be make into two kinds of issues
    1. Many issues to change the easy alt text on the page
    2. one Large issue that adds the aria text to the svgs and figures out how to keep them from the code when see more is clicked (if its even possible).

**Comment by ExperimentsInHonesty** [comment]
Currently, if you go to the wins page, the alt text for the icons is the name of the file. That needs to be changed to the related text

so for the briefcase it says alt="briefcase.svg" and it should says alt="I got a new job"

 line 23 of this file: https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js
  const badgeIcons = {
		"I got a new job": `briefcase.svg`,

However, when you go to the "see more" version of the page, the alt text should change to alt="" because the words are appearing next to the icons

![image](https://user-images.githubusercontent.com/37763229/214198667-3c24c659-938b-4318-9dc9-286fc9669a4b.png)

**Comment by roslynwythe** [comment]
Hi @ExperimentsInHonesty  I agree that there are two separate objectives: 1) on the page as it initially loads, place meaningful alt text on the badge icons, and 2) on the modal overlay ("See More"), to remove the alt text. However in both cases, the html is  generated by the same lines of JS code, in particular, line 266 and 273 of https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js so I think I just need to create one large issue. 

- Follow-up issue: #3960

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe The original reason why two issues needed to be created was because there are other images beside the achievement badge icons.  And those might still need their alt text changes.  Can you check that out?

**Comment by roslynwythe** [comment]
Yes @ExperimentsInHonesty I see now. I apologize for just focusing on the badge icons.  I believe these are all functional, not decorative, and will require alt text: 

- the quote
- LinkedIn profile
- GitHub profile
- GitHub image.  

I assume that the alt text should be personalized, so for instance the alt text for the GitHub image on Shruti's card would read "Github profile picture for Shruti Mukherjee" and the quote would read "Open quote from Shruti Mukherjee".  

- Here is the first issue (draft):  #4010
If you think its ok I'll create the others but in order to avoid merge conflicts, should I make the subsequent issues dependent on the others, so only one is in progress at a time?

I used the chrome screen reader extension as you suggested and I was shocked at how little information was provided to the user.  Screen readers rely on a descriptive title to give the user an idea of what the page is about, and so  I suggest changing the Wins page title from "Wins - Hack for LA" to "Celebrate our Wins" or "Wins: Let's celebrate our contributors' accomplishments".   I was surprised that the screen reader did not read the h1 heading.   I wonder if we should add some aria tags, to the h1 or to the section elements.

**Comment by roslynwythe** [comment]
Follow-up issues: 
- #4010 
- #4088 
- #4112 
- #4113
